e2e2/internal/framework_extensions/helm.go

```diff
@@ -0,0 +1,65 @@
+package frameworkext
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/getter"
+	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/repo"
+)
+
+func InstallPackagedRelease(repoName string, repoUrl string, chart string, releaseName string, namespace string,
```

Comment:
Can we reuse any of the helm support from the framework instead of rolling our own? https://github.com/kubernetes-sigs/e2e-framework/blob/c61338a3773c2b64550d0e46037bfe541fa54645/third_party/helm/helm.go

Reply:
Updated